### PR TITLE
Feature previous msg

### PR DESCRIPTION
**Pull request by Kit486759**
Hi, I have created a feature to scroll and show previous messages in chatbox. Please have a look and message me if any issues.
There are some comment code, will clean up before merge.

### CODE REVIEW

File: src/components/MusicianPage/chat/ChatScreen.js [hunk 1]
```diff
@@ -69,11 +74,48 @@ const ChatScreen = ({ uid, roomId, otherProfile }) => {
     }
   };
 
+  useEffect(
+    () => {
+      const messageBox = document.getElementsByClassName("messages-body")[0];
```
Comment (BlackStone8960) [inline src/components/MusicianPage/chat/ChatScreen.js:79]: Can you use useRef to get elements (in React)?

**Review by BlackStone8960 — COMMENTED** [review]
Thank you so much, it's much better than before. 
But you are loading 15, 30, 45... messages after user scrolls for now, so the data traffic can be very large with current code. Can you make it load just additional 15 messages after each scrolling and if you can? 
If not, it's fine with current code as well, so let me know ASAP. Thank you so much!

**Review by miyabitanimchi — COMMENTED** [review]
@Kit486759 great work, but I'd just like to know if you also tried to use the endBefore() method from Firebase and if it worked. Did you try that?

**Comment by YumiKimbara** [comment]
I think it works without any problem!

**Comment by Kit486759** [comment]
> Thank you so much, it's much better than before. But you are loading 15, 30, 45... messages after user scrolls for now, so the data traffic can be very large with current code. Can you make it load just additional 15 messages after each scrolling if you can? If not, it's fine with current code as well, so let me know ASAP. Thank you so much!

Um, I am not sure if I can, let me try it.

**Comment by Kit486759** [comment]
> @Kit486759 great work, but I'd just like to know if you also tried to use the endBefore() method from Firebase and if it worked. Did you try that?

No I didn't , I will have a look on it. Thank you

**Review by BlackStone8960 — APPROVED** [review]
Hi Kit, can you increase the range of loading messages to 30 at each time? I mean we can display 30 messages first, then load 60 messages, 90, 120...

**Comment by Kit486759** [comment]
> Hi Kit, can you increase the range of loading messages to 30 at each time? I mean we can display 30 messages first, then load 60 messages, 90, 120...

 Hi Taichi, I have changed the message qty to 30, Please review . Thank you!

**Review by BlackStone8960 — APPROVED** [review]
Good job! Thank you so much

**Pull request merged**